packages/x-components/src/plugins/devtools/wiring.devtools.ts

```diff
@@ -0,0 +1,116 @@
+import { CustomInspectorNode, DevtoolsPluginApi, InspectorNodeTag } from '@vue/devtools-api';
+import { forEach, reduce } from '../../utils/object';
+import { XEvent } from '../../wiring/events.types';
+import { filter } from '../../wiring/wires.operators';
+import { AnyWire, Wiring } from '../../wiring/wiring.types';
+import { XModuleName } from '../../x-modules/x-modules.types';
+import { moduleColors } from './utils';
+
+/** The full list of wiring nodes for the inspector. */
+const WiringNodes: Partial<Record<XEvent, CustomInspectorNode[]>> = {};
+/** Status of the nodes. */
+const WiresStatus: Record<string, boolean> = {};
+
+/**
+ * Creates tags given a node representing a wire.
+ *
+ * @param node - The inspector node representing a {@link Wire}.
+ * @returns A list of tags for the given node.
+ */
+function createWireTags({ tags = [], id }: CustomInspectorNode): InspectorNodeTag[] {
+  const newTags: InspectorNodeTag[] = [...tags];
+  if (!WiresStatus[id]) {
+    newTags.push({ label: 'disabled', backgroundColor: 0x900909, textColor: 0xfbd0d0 });
```

Comment:
We've a lot of hex literal colors around the pr without explanation, I know it's not a big deal just to be perfect 😗